src/saltedhash.h

```diff
@@ -50,4 +50,28 @@ class SaltedOutpointHasher
     }
 };
 
+class SaltedIDHasher
+{
+private:
+    /** Salt */
+    uint64_t k0, k1;
+
+public:
+    SaltedIDHasher();
+
+    size_t operator()(const uint160& id) const;
+};
+
+class SaltedScriptHasher
+{
+private:
+    /** Salt */
+    const uint64_t k0, k1;
```

Comment:
28dddb91b9744987125cd2248577f38c0b2e77c7
```suggestion
    const uint64_t m_k0, m_k1;
```

Reply:
Done